mappings/net/minecraft/item/Items.mapping

```diff
@@ -237,6 +237,7 @@ CLASS net/minecraft/class_1802 net/minecraft/item/Items
 	FIELD field_27075 CUT_COPPER Lnet/minecraft/class_1792;
 	FIELD field_27076 LIGHTLY_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
 	FIELD field_27077 SEMI_WEATHERED_CUT_COPPER Lnet/minecraft/class_1792;
+	FIELD field_27876 POWDER_SNOW_BUCKET Lnet/minecraft/class_1792;
```

Comment:
It's automapped for me

Reply:
Yep. Aren't the mappings for `Item` instances only for blocks since they don't get automapped?